### PR TITLE
Sci-Fi Fair Blog Post

### PR DESCRIPTION
**Pull request by parkan**
Sci-Fi Fair blog post. A few notes:

- opted for direct video embeds over thumbnails + links for the draw-in videos to create more engagment
- sizzle reel somewhat embarrassingly opens with my name
- needs links for names etc, to come
- needs final call to action section
- needs polish on copy and likely copyediting/typos

Mainly looking for comments on overall structure and ideas

### CODE REVIEW

**Comment by momack2** [comment]
This is PHENOMENAL!!! Love love love it! Will let you take a pass on polish first but ping when want a final review!

File: content/post/059-ipfs-camp-sci-fi-fair-videos.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+![@gorhgorh's catbot](/059-ipfs-camp-sci-fi-fair-videos/catbot.jpg)
+_Jérôme Loï's libp2p-powered Lazer Catbot (unfortunately no cats were on premises)_
+
+![the elusive @rklaehn](/059-ipfs-camp-sci-fi-fair-videos/actyx.jpg)
+_Actyx's Rüdiger has somehow managed to avoid ALL photos at the fair, so here's a shot of him demoing their amazing industrial automation system from the lightning talks_ 
+
+![@aschmahmann's Fast IPNS](/059-ipfs-camp-sci-fi-fair-videos/ipns.jpg)
+_Last but not least, Adin's "Fast IPNS" demo (it was VERY FAST!)_
+
+## Community Interviews
```
Comment (momack2) [inline content/post/059-ipfs-camp-sci-fi-fair-videos.md]: @doctorrobinson - can you add a few paragraphs highlighting the community interviews (maybe calling out Netflix and Audius like Arkadiy's examples above)?

File: content/post/059-ipfs-camp-sci-fi-fair-videos.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+IPFS Camp was all about YOU so the interviews weren't limited to just the Sci-Fi Fair participants: we managed to get some async sit-down time with a few other wonderful community members: see their accounts in [this playlist](TK)
```
Comment (doctorrobinson) [inline content/post/059-ipfs-camp-sci-fi-fair-videos.md]: IPFS Camp was all about YOU so the interviews weren't limited to just the Sci-Fi Fair participants: we managed to get some sit-down time with a few other wonderful community members to talk about their projects. Check out the videos below or skip right to the [playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTwwGv30nKdhr3GiIAnhbyE). 

[Netflix Containers - Edgar Lee](https://youtu.be/wNfk05D887M) 
We hear from Edgar about how he's exploring distributing container layers through IPFS, and how it's changing his workflow at Netflix. 

[Audius - Hareesh Nagaraj](https://youtu.be/3pSV6ai6QUI) 
We speak to Hareesh from Audius, a decentralized music streaming platform, about how they're using IPFS as their primary file storing system. 

[Radicle & Monadic - Julian Arni](https://youtu.be/UCDHMKC8Seg) 
Julian tells us about his peer-to-peer collaboration platform Radicle and speaks about the benefits of decentralized systems. 

[World Brain - Oliver Sauter](https://youtu.be/e1YfSa2_Drc) 
We talk to Oliver about their decentralized search tool, Memex, and their peer-to-peer search engine for personal and collective knowledge, Worldbrain.io. 

[IPFS Force - Steven Li](https://youtu.be/KsMqesFi4Ac) 
And on top of all that, Steven joins us from Shanghai City to discuss his work on IPFS-Force.

File: content/post/059-ipfs-camp-sci-fi-fair-videos.md (hunk 1 above)
Comment (doctorrobinson) [reply content/post/059-ipfs-camp-sci-fi-fair-videos.md]: Dropped it below - @parkan can you add to your update please?

File: content/post/059-ipfs-camp-sci-fi-fair-videos.md (hunk 2 above)
Comment (momack2) [reply content/post/059-ipfs-camp-sci-fi-fair-videos.md]: Zak - I'm moving these to their own post. They're so great and I feel like after all the awesomeness of the scifi fair they are getting a little lost down here...

**Comment by parkan** [comment]
ok ready for review and scheduled for tomorrow (Oct 3)

**Comment by parkan** [comment]
@doctorrobinson @momack2 I don't have write on this repo, can you merge?

**Comment by doctorrobinson** [comment]
Thanks for driving this over the finish line while I was out last week!

**Comment by parkan** [comment]
hmm I don’t see this on the blog despite the number and the date seemingly being correct?

https://circleci.com/gh/ipfs/blog/722 looks good overall though I don’t see the supposed built asset

**Comment by doctorrobinson** [comment]
I saw the same thing yesterday  - I think this is outside of my wheelhouse for things I can fix, maybe @daviddias can help us troubleshoot?

**Comment by daviddias** [comment]
@autonome got there first with https://github.com/ipfs/blog/issues/312

**Pull request merged**